modules/installation-vsphere-infrastructure.adoc

```diff
@@ -59,7 +59,7 @@ You must install the {product-title} cluster on a VMware vSphere version 6 or 7
 
 [IMPORTANT]
 ====
-Installing a cluster on VMware vSphere version 6.7U2 or earlier and virtual hardware version 13 is now deprecated. These versions are still fully supported, but support will be removed in version 4.11 of {product-title}. Hardware version 15 is now the default for vSphere virtual machines in {product-title}. To update the hardware version for your vSphere nodes, see the "Updating hardware on nodes running in vSphere" article.
+Installing a cluster on VMware vSphere version 6.7U3 or earlier and virtual hardware version 13 is now deprecated. These versions are still fully supported, but version 4.11 of {product-title} will require vSphere version 7.0 or later. Hardware version 15 is now the default for vSphere virtual machines in {product-title}. To update the hardware version for your vSphere nodes, see the "Updating hardware on nodes running in vSphere" article.
```

Comment:
We're still going to support vSphere 6.7U3 in 4.11 in alignment with Kubernetes 1.24.

Change "These versions are still fully supported, but version 4.11 of {product-title} will require vSphere version 7.0 or later." to "These versions are still fully supported, but version 4.11 of {product-title} will require vSphere virtual hardware version 15 or later."